homeassistant/components/cloud/cloudhooks.py

```diff
@@ -14,6 +14,9 @@ def __init__(self, cloud):
 
     async def async_publish_cloudhooks(self):
         """Inform the Relayer of the cloudhooks that we support."""
+        if not self.cloud.is_connected:
+            return
```

Comment:
Should we log something in this case (why would we not be connected yet invoking publish?)  Also I didn't see any test cases for this condition.

Reply:
Nah, because this will get called often by integrating components and they shouldn't have to check the cloud status themselves first IMHO.

Reply:
Makes sense!

Reply:
A cloud hook can be created even when we're not connected to the relayer.